### PR TITLE
Issue #12 started integrating Thymeleaf into the application

### PR DESCRIPTION
**Pull request by artemlisikh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,6 +68,16 @@
...
+        <dependency>
+            <groupId>org.thymeleaf</groupId>
+            <artifactId>thymeleaf</artifactId>
+            <version>3.0.6.RELEASE</version>
+        </dependency>
+		<dependency>
+			<groupId>org.thymeleaf</groupId>
+			<artifactId>thymeleaf-spring4</artifactId>
+			<version>3.0.6.RELEASE</version>
+		</dependency>
```
Comment (olisikh) [inline pom.xml]: Please fix formatting :)

CTRL + ALT + L in Intellij IDEA - could be triggered on a file or on selected text.

**Review by olisikh — COMMENTED** [review]
Great change. Now we can start coding the UI part

File: pom.xml (hunk 1 above)
Comment (artemlisikh) [reply pom.xml]: Didn't notice that

**Review by olisikh — APPROVED** [review]
Well done!

**Pull request merged**